src/node_process_methods.cc

```diff
@@ -511,7 +511,7 @@ static void LoadEnvFile(const v8::FunctionCallbackInfo<v8::Value>& args) {
 
   switch (dotenv.ParsePath(path)) {
     case dotenv.ParseResult::Valid: {
-      dotenv.SetEnvironment(env);
+      USE(dotenv.SetEnvironment(env));
```

Comment:
Shouldn't we just throw here, rather than omitting the error?

Reply:
We're not ignoring the error. In this branch we end up immediately breaking and skipping to the end of the function, allowing the error to propagate.